Author: Stephen Baynes <stephen.baynes@smoothwall.net>
Description: A rather hacky workround for the combination of squid bugs
  http://bugs.squid-cache.org/show_bug.cgi?id=4416
  http://bugs.squid-cache.org/show_bug.cgi?id=4417

  Stop the attempt by the coordinator process to open a non existant
  cache being fatal. If a guenuine cache file is not openable then
  higher up checks also catch the issue and make it fatal so this is safe
  should there be a real error. Plenty of clear debug in log file when this
  happens.
--- a/src/fs/ufs/UFSSwapDir.cc
+++ b/src/fs/ufs/UFSSwapDir.cc
@@ -729,7 +729,7 @@
 
     if (swaplog_fd < 0) {
         debugs(50, DBG_IMPORTANT, "ERROR opening swap log " << logPath << ": " << xstrerror());
-        fatal("UFSSwapDir::openLog: Failed to open swap log.");
+        return;
     }
 
     debugs(50, 3, HERE << "Cache Dir #" << index << " log opened on FD " << swaplog_fd);
